service/controller/chartconfig/v7/resource_set.go

```diff
@@ -14,10 +14,10 @@ import (
 	"github.com/spf13/afero"
 	"k8s.io/client-go/kubernetes"
 
-	"github.com/giantswarm/chart-operator/service/controller/chartconfig/v6/key"
-	"github.com/giantswarm/chart-operator/service/controller/chartconfig/v6/resource/chart"
-	"github.com/giantswarm/chart-operator/service/controller/chartconfig/v6/resource/chartstatus"
-	"github.com/giantswarm/chart-operator/service/controller/chartconfig/v6/resource/tiller"
+	"github.com/giantswarm/chart-operator/service/controller/chartconfig/v7/key"
+	"github.com/giantswarm/chart-operator/service/controller/chartconfig/v7/resource/chart"
+	"github.com/giantswarm/chart-operator/service/controller/chartconfig/v7/resource/chartstatus"
```

Comment:
Further fixing the importing

Reply:
Do you maybe want to check all the imports and provide a separate PR for this so we get over it now? It looks to me now that this is not some single accident but that we apparently missed to do this properly.

Reply:
Sure, I will make separate PR for it.

Reply:
PR created: https://github.com/giantswarm/chart-operator/pull/254 I will wait until it merged first.